numpy/linalg/lapack_lite/make_lite.py

```diff
@@ -281,6 +285,44 @@ def ensure_executable(name):
     except:
         raise SystemExit(name + ' not found')
 
+def create_name_header(output_dir):
+    routine_re = re.compile(r'^      (subroutine|.* function)\s+(\w+)\(.*$',
+                            re.I)
+    extern_re = re.compile(r'^extern [a-z]+ ([a-z0-9_]+)\(.*$')
+
+    # BLAS/LAPACK symbols
+    symbols = set(['xerbla'])
+    for fn in os.listdir(output_dir):
+        fn = os.path.join(output_dir, fn)
+
+        if not fn.endswith('.f'):
+            continue
+
+        with open(fn, 'r') as f:
+            for line in f:
+                m = routine_re.match(line)
+                if m:
+                    symbols.add(m.group(2).lower())
+
+    # f2c symbols
+    f2c_symbols = set()
+    with open('f2c.h', 'r') as f:
+        for line in f:
+            m = extern_re.match(line)
+            if m:
+                f2c_symbols.add(m.group(1))
+
+    with open(os.path.join(output_dir, 'lapack_lite_names.h'), 'w') as f:
+        f.write(HEADER_BLURB + "\n")
```

Comment:
Mind adding a brief comment about what this file is for to the top of the file, just below this blurb?

Reply:
Added.